BlackjackGame/shared/src/model/Card.java

```diff
@@ -0,0 +1,49 @@
+public class Card{
+	
+	private enum CardSuit{
+		Spades,
+		Hearts,
+		Clubs,
+		Diamonds;
+	}
+	
+	private enum CardRank{
+		Ace, Two, Three, Four, Five, Six, Seven, Eight, Nine, Ten, Jack, Queen, King
+	}
```

Comment:
can you add the enums as their own public files? 
```
 shared  ->
        model
                Suit
                Rank 
``` 
That way when we need to determine values or render the cards on screen we have acces to the enums.